src/client/client4.js

```diff
@@ -2736,7 +2742,12 @@ export default class Client4 {
             return;
         }
 
-        const properties = Object.assign({category, type: event, user_actual_id: this.userId}, props);
+        const properties = Object.assign({
+            category,
+            type: event,
+            user_actual_role: this.userRoles && isSystemAdmin(this.userRoles) ? 'system_admin' : null,
```

Comment:
Probably here we will need in the future something more clear to include the guest users. So I would set here it to 'system_admin' or 'system_user' for now, and in the future we can return 'system_admin', 'system_user' or 'system_guest'. (The "system" is important here, because we are only talking about the role at system level, not the role in the context of the team or channel).

Reply:
I agree, but the ticket specifically asked to only include the system_admin flag...

Reply:
Yes, you are right, this was explicitly design that way @jasonblais what do you think about my proposed change?

Reply:
Done!

Reply:
done!

Reply:
We're modelling data here, not the UI, so IMO we should be using the internal terms of `sytem_admin` and `system_user`. Grepping for either shows lots of context in `mattermost-server`. Grepping for `system_member` shows nothing. 

Whatever interprets this data downstream is free to render it as "system_member" if it makes more sense to those consuming it, but I think it will be easier to work with if we maintain consistency.

Reply:
(GitHub has somehow screwed up the comment order here -- my reply above was definitely /after/ @jasonblais's comment at the bottom.)

Reply:
@jespino , @jasonblais - do we have a consensus on the value? system_member vs system_user

Reply:
@lieut-data I think I agree with you about it, but I get the @jasonblais point which is, the analytics data is not something consumed by the developers, normally is somehow user facing data. I don't have an strong opinion here, for me, makes more sense the `system_user` but I'm very biased by the fact that is what I see all the time in the code.

Reply:
What's consistent with other diagnostics data? Do we typically follow the internal term, or the external (UI) term?

Reply:
I think is not super consistent in that sense, so... I feel right either way, adding this consistent with the system data or use it as user facing data.

Reply:
Question: Do System Admins inheirt both the `system_user` and `system_admin` role, or is it only the `system_admin` role?

Reply:
> Both

Then I'm wondering how you all feel about this proposal:

1. If System Admin, mark the user's role as `system_admin, system_user`
2. Else, mark the user's role as `system_user`

Pros:
1. Follows our data model (obvious for a developer audience)
2. Follows the convention used in the UI, where "everyone in the system" is considered a user (obvious for a data consumer audience)
3. Scaleable, as we might add more roles in future (think of team_admin, channel_admin) where we'd want to support a list of roles the user has inherited.

@reflog @lieut-data @jespino

Reply:
I like it too